modules/theme-tools/content-options/customizer.php

```diff
@@ -404,6 +404,27 @@ public function render_content() {
 				)
 			);
 		}
+		
+		// Featured Images: All other custom post types
+		$wp_customize->add_setting(
+			'jetpack_content_featured_images_custom',
+			array(
+				'default'           => '',
+				'type'              => 'option',
+				'sanitize_callback' => 'wp_filter_nohtml_kses',
+			)
+		);
+
+		$wp_customize->add_control(
+			'jetpack_content_featured_images_custom',
+			array(
+				'section'         => 'jetpack_content_options',
+				'label'           => esc_html__( 'Hide on these custom post types', 'jetpack' ),
+				'description'	  => esc_html__( 'Enter comma separated list of custom post type slugs, where you want to hide featured images. Will display on all others by default', 'jetpack' ),
```

Comment:
As I noted in the original issue, it may be best to provide a better interface for this:

> I would instead recommend a list of checkboxes listing all publicly displayed post types on the site, so it's easier for one to check or uncheck where they want featured images.